test/e2e/pkg/manifest.go

```diff
@@ -160,8 +160,8 @@ type ManifestNode struct {
 	// this relates to the providers the light client is connected to.
 	PersistentPeers []string `toml:"persistent_peers"`
 
-	// Database specifies the database backend: "goleveldb", "cleveldb",
-	// "rocksdb", "boltdb", "pebbledb" or "badgerdb". Defaults to goleveldb.
+	// Database specifies the database backend: "goleveldb", "rocksdb",
```

Comment:
again, is `goleveldb` supported in e2e ?  just checking again based on content of Dockerfile